incubator/jenkins-operator/templates/NOTES.txt

```diff
@@ -0,0 +1,8 @@
+1.  jenkins-operator deployed.
+
+    Check the jenkins-operator logs:
+        export POD=$(kubectl get pods -l app={{ template "jenkins-operator.name" . }} --namespace {{ .Release.Namespace }} --output name)
+        kubectl logs $POD --namespace={{ .Release.Namespace }}
+
+2.  If want to deploy a jenkins instance, please create Custom Resource objects, as described in jenkins operator documentation
```

Comment:
Nice to have:

Add a sample CRD definition and apply command here so that users don't to go back to the docs to kick the tires.